app/src/main/java/com/eveningoutpost/dexdrip/utils/DexCollectionType.java

```diff
@@ -78,6 +79,7 @@
         Collections.addAll(usesBattery, BluetoothWixel, DexbridgeWixel, WifiBlueToothWixel, WifiDexBridgeWixel, Follower, LimiTTer, LibreAlarm, LimiTTerWifi, LibreWifi); // parakeet separate
         Collections.addAll(usesDexcomRaw, BluetoothWixel, DexbridgeWixel, WifiWixel, WifiBlueToothWixel, DexcomG5, WifiDexBridgeWixel, Mock);
         Collections.addAll(usesTransmitterBattery, WifiWixel, BluetoothWixel, DexbridgeWixel, WifiBlueToothWixel, WifiDexBridgeWixel); // G4 transmitter battery
+        Collections.addAll(supportsSensorStartStop, BluetoothWixel, DexbridgeWixel, LimiTTer, LimiTTerWifi, LibreWifi, WifiBlueToothWixel, WifiWixel, DexcomG5, DexcomG6, WifiDexBridgeWixel, LibreAlarm, Medtrum, LibreReceiver);
```

Comment:
@jamorham Which implementation really supports sensor starts/stops? I believe it is only `DexcomG5`, `DexcomG6` and subset or all entries of `usesLibre`.